CondTools/Ecal/python/copyBadStrip_cfg.py

```diff
@@ -39,11 +40,12 @@
     IsDestDbCheckedInQueryLog=cms.untracked.bool(True),
     SinceAppendMode=cms.bool(True),
     Source=cms.PSet(
-     firstRun = cms.string('120000'),
+     firstRun = cms.string('200000'),
      lastRun = cms.string('10000000'),
      OnlineDBSID = cms.string('cms_omds_lb'),
-     OnlineDBUser = cms.string('cms_ecal_conf'),
-     OnlineDBPassword = cms.string('*************'),
+#     OnlineDBSID = cms.string('cms_orcon_adg'),  test on lxplus
+     OnlineDBUser = cms.string('cms_ecal_r'),
+     OnlineDBPassword = cms.string('3c4l_r34d3r'),
```

Comment:
Hi,

in April I did a PR-13967 for 8_1_X in which all the passwords (16 files)  were replaced by *** and for which the DB access was an offline  (lxplus) one.
This version was merged  on 2 May and is active in 8_1_0_pre9.

For O2O online, if you want a 8_0_X version, I have updated PR-15319 with the copy of what is online on O2O machine in 8_0_1.  As Giacomo wanted that the change of Ecal O2O would be transparent, it is the only way to do it (Password active and access to online DB).

So either you backport what is on 8_1_X (PR-13967) to  8_0_X for public software (without password and offline DB) but this version is not correct for future versions of O2O, either you consider the PR-15319 (to be propagated to 8_1_X) to be correct immediately for future O2O versions, but with passwords readable (mainly on read-only DB).  I don't see intermediate solution.

Cheers.

Pierre


________________________________
From: Marco Musich [notifications@github.com]
Sent: 02 August 2016 09:26
To: cms-sw/cmssw
Cc: Pierre Depasse; Mention
Subject: Re: [cms-sw/cmssw] O20 for 8_0_15 (#15319)


In CondTools/Ecal/python/copyBadStrip_cfg.py<https://github.com/cms-sw/cmssw/pull/15319#discussion_r73105810>:

>       lastRun = cms.string('10000000'),
>       OnlineDBSID = cms.string('cms_omds_lb'),
> -     OnlineDBUser = cms.string('cms_ecal_conf'),
> -     OnlineDBPassword = cms.string('*************'),
> +#     OnlineDBSID = cms.string('cms_orcon_adg'),  test on lxplus
> +     OnlineDBUser = cms.string('cms_ecal_r'),
> +     OnlineDBPassword = cms.string('3c4l_r34d3r'),


@depasse<https://github.com/depasse> I don't think putting the ecal account password in clear in the release code is good practice

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/cms-sw/cmssw/pull/15319/files/4b5d7b61df01cfc342437343866f92078abccab0#r73105810>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AEaCWyH7oNP44RAgpOZYMFZcA60JbY3Pks5qbvEVgaJpZM4JXVmQ>.